packages/rancher-sriov/generated-changes/patch/templates/clusterrole.yaml.patch

```diff
@@ -0,0 +1,54 @@
+--- charts-original/templates/clusterrole.yaml
++++ charts/templates/clusterrole.yaml
+@@ -52,3 +52,51 @@
+   - apiGroups: [""]
+     resources: ["pods/eviction"]
+     verbs: ["create"]
++---
++apiVersion: rbac.authorization.k8s.io/v1
++kind: ClusterRole
++metadata:
++  name: sriov-admin
++  {{- if .Values.global.rbac.userRoles.aggregateToDefaultRoles }}
++    rbac.authorization.k8s.io/aggregate-to-admin: "true"
++  {{- end }}
++rules:
++- apiGroups:
++  - sriovnetwork.openshift.io
++  resources:
++  - '*'
++  verbs:
++  - '*'
++---
++apiVersion: rbac.authorization.k8s.io/v1
++kind: ClusterRole
++metadata:
++  name: sriov-edit
++  {{- if .Values.global.rbac.userRoles.aggregateToDefaultRoles }}
```

Comment:
Does a default need to be patched in the values.yaml? e.g.

```yaml
global:
  rbac:
    userRoles:
      aggregateToDefaultRoles: true|false
```

If a `helm template` does not cause a failure and the default for this is expected to be false, then we can continue to omit this. But I'm not sure what would happen since `global.rbac.userRoles` is undefined right now.

Reply:
Yes, we need this

Reply:
Sounds good.